bust-a-move-framework/src/main/java/bustamove/bubble/BubbleStorage.java

```diff
@@ -191,14 +193,9 @@ public boolean isFull() {
      */
     public void removeBubble(final Bubble bubble) {
         for (Bubble[] row : bubble2DArray) {

```

Comment:
it isnt much of a performance hit if we let the entire loop run. I think this is fine as is